### DISCUSSION TITLE
RuntimeError: cuDNN error: CUDNN_STATUS_INTERNAL_ERROR

### DISCUSSION THREAD

**Discussion by Mgithus** [Project-MONAI/MONAI#6853] [Q&A]
**Describe the bug**

**To Reproduce**
2. Install '....'
3. Run commands '....'

**Expected behavior**

**Screenshots**

**Environment**

Ensuring you use the relevant python executable, please paste the output of:

```
python -c 'import monai; monai.config.print_debug_info()'
```

**Additional context**

**Comment by KumoLiu** [comment]
Hi @Mgithus, hope this can help you.
https://stackoverflow.com/questions/62067849/pytorch-model-training-runtimeerror-cudnn-error-cudnn-status-internal-error
Thanks!
